yt_dlp/extractor/digiview.py

```diff
@@ -0,0 +1,112 @@
+import urllib.parse
+
+from yt_dlp.utils import int_or_none
+
+from .youtube import YoutubeIE
+from ..networking import Request
+
+
+class DigiviewIE(YoutubeIE):
+    IE_DESC = 'Digiview'
+    IE_NAME = 'digiview'
+    _VALID_URL = r'https?://(?:www\.)?ladigitale\.dev/digiview/#/v/(?P<id>[0-9a-f]+)'
+    _TESTS = [
+        {
+            # normal video
```

Comment:
This video has since been removed from YouTube. Could you please update the tests?